tabbycat/draw/generator/common.py

```diff
@@ -170,7 +171,7 @@ class BasePairDrawGenerator(BaseDrawGenerator):
         "avoid_conflicts"       : "off",
     }
 
-    TEAMS_PER_DEBATE = 2
+    teams_in_debate = 2
```

Comment:
I think if you set `self.teams_in_debate` to an instance attribute in the constructor (line 48), that takes precedence over the class attribute (here), right? So this effectively now does nothing? (I see you deleted it from `BaseBPDrawGenerator`.)